denys/Lesson01-Favorite-Toys/T01.01-Exercise-CreateLayout/gradlew

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
+
+##############################################################################
+##
+##  Gradle start up script for UN*X
```

Comment:
[This answer](http://stackoverflow.com/a/39249309) explains why you would want to use the gradle wrapper instead of the local gradle